packages/lambdas/src/fhir-to-bundle.ts

```diff
@@ -0,0 +1,86 @@
+import { Resource } from "@medplum/fhirtypes";
+import { medical } from "@metriport/shared";
+import { SearchSetBundle } from "@metriport/shared/medical";
+import { Patient } from "@metriport/core/domain/patient";
+import { getConsolidatedFhirBundle } from "@metriport/core/external/fhir/consolidated";
+import { out } from "@metriport/core/util/log";
+import * as Sentry from "@sentry/serverless";
+import { capture } from "./shared/capture";
+import { getEnvOrFail } from "./shared/env";
+import { apiClient } from "./shared/oss-api";
+
+// Keep this as early on the file as possible
+capture.init();
+
+const lambdaName = getEnvOrFail("AWS_LAMBDA_FUNCTION_NAME");
+const apiURL = getEnvOrFail("API_URL");
+
+const ossApi = apiClient(apiURL);
+
+export const handler = Sentry.AWSLambda.wrapHandler(
+  async ({
+    patient,
+    requestId,
+    conversionType,
+    documentIds,
+    resources,
+    dateFrom,
+    dateTo,
+    isAsync,
+  }: {
+    patient: Pick<Patient, "id" | "cxId">;
+    requestId?: string;
+    conversionType?: medical.ConsolidationConversionType;
+    documentIds?: string[];
+    resources?: medical.ResourceTypeForConsolidation[];
+    dateFrom?: string;
+    dateTo?: string;
+    isAsync?: boolean;
+  }): Promise<SearchSetBundle<Resource> | void> => {
+    const { log } = out(`cx ${patient.cxId}`);
+    log(
+      `Running with dateFrom: ${dateFrom}, dateTo: ${dateTo}, ` +
+        `documentIds: ${documentIds}, resources: ${resources}}`
+    );
+    try {
+      const bundle = await getConsolidatedFhirBundle({
+        patient,
+        documentIds,
+        resources,
+        dateFrom,
+        dateTo,
+      });
+      if (isAsync) {
+        ossApi.postConsolidated({
```

Comment:
All code in the lambda should be synchronous because we want to make sure the code is completed before the lambda environment is teared down upon exit of the handle function